src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -185,6 +189,35 @@ private function extractFromAccessor(string $class, string $property): ?array
         return null;
     }
 
+    /**
+     * Tries to extract type information from constructor.
+     *
+     * @return Type[]|null
+     */
+    private function extractFromConstructor(string $class, string $property): ?array
+    {
+        try {
+            $class = new \ReflectionClass($class);
+            $constructor = $class->getConstructor();
+        } catch (\ReflectionException $e) {
+            return null;
+        }
+
+        if (!$constructor) {
+            return null;
+        }
+
+        foreach ($constructor->getParameters() as $name => $parameter) {
```

Comment:
Is the `$name` of the foreach different than `$parameter->name` or we can reuse it?

Reply:
`$name === $string` (note the leading `$`).

So I removed the `$name` assignment. Thanks